src/_sass/tilt.scss

```diff
@@ -132,6 +132,8 @@ $on-docs-mobile:    $header-width + $spacing-unit;
 
 // --------------------------------------------------
 
+$z-modal: 1500;
```

Comment:
FYI, I used Sass vars for z-index so we can see this single list of how various elements are stacked up!